src/radio/handlers.js

```diff
@@ -2,36 +2,34 @@
 
 import {Writable} from 'stream';
 import type {ConfigOptions} from '..';
-import soundcloud from './handlers/soundcloud';
-import youtube from './handlers/youtube';
+import SoundCloud from './handlers/soundcloud';
+import YouTube from './handlers/youtube';
 
-const handlers: Array<Class<Handler>> = [soundcloud, youtube];
+const handlers: * = [SoundCloud, YouTube];
 
 export interface Handler {
-  // constructor(link: string, config: $PropertyType<ConfigOptions, 'services'>): Handler;
+  // constructor(link: string, config: $PropertyType<ConfigOptions, 'services'>);
   // static match(link: string): boolean;
-  getMeta(cb: (error: ?Error, song?: SongInfo) => void): void;
+  getMeta(): Promise<SongInfo>;
   download(stream: Writable): Writable;
 }
 
-export interface SongInfo {
-  id: string;
-  title: string;
-  url: string;
-  image: string;
-  duration: number; // in seconds
-  plays: string;
+export type SongInfo = {
```

Comment:
Wasn't this originally a type and then we changed it when swapping to flow? Why the decision to change this?

Reply:
Now that we both have a bit more experience with Flow's `interface` vs `type`, I'm moving these over. Interfaces are intended to be implemented (by classes). OTOH, these are just typings on some objects.

Changing them also surfaced errors where these were being imported without `import type`, which is also a bonus to me. These types of definitions shouldn't be getting imported like everything else, because that just gets confusing.